rakudo-star/content.md

```diff
@@ -1,16 +1,14 @@
 # What is Rakudo Star?
 
-Rakudo Star is a Perl 6 distribution designed for use by early adopters of the language. It includes a virtual machine (the JVM or MoarVM), the Rakudo Perl 6 compiler, and a suite of modules that users may find useful. This image includes the MoarVM backend for the compiler.
+Rakudo Star is a Raku distribution designed for use by early adopters of the language. It includes a virtual machine (the JVM or MoarVM), the Rakudo compiler, and a suite of modules that users may find useful. This image includes the MoarVM backend for the compiler.
 
 Project homepage: [http://rakudo.org](http://rakudo.org)
 
 GitHub repository: [https://github.com/rakudo/star](https://github.com/rakudo/star)
 
-The Dockerfile responsible: [http://github.com/perl6/docker/tree/master/Dockerfile](http://github.com/perl6/docker/tree/master/Dockerfile)
+The Dockerfile responsible: [http://github.com/raku/docker/tree/master/Dockerfile](http://github.com/raku/docker/tree/master/Dockerfile)
 
-Perl 6 Language Specification: [http://design.perl6.org/](http://design.perl6.org/)
-
-Perl 6 Language Documentation: [http://doc.perl6.org/](http://doc.perl6.org/)
+Perl 6 Language Documentation: [http://docs.raku.org/](http://docs.raku.org/)
```

Comment:
There's a few instances of "Perl 6" in here; did you want to update them as well, or leave them as-is?

Reply:
Good catch, replaced them in another commit.